.cspell.json

```
@@ -1385,6 +1386,8 @@
     "rsplit",
     "rstar",
     "rsyslog",
+    "Rtklib",
+    "rtklib",
```

Same here. Also, FYI, it's case-insensitive.

```suggestion
    "RTKLIB",
```